CMake/cdat_modules/cmor_pkg.cmake

```diff
@@ -5,4 +5,8 @@ set(CMOR_MD5 f7e78104878d93ffbd3cae2ccd95db2c)
 set (nm CMOR)
 string(TOUPPER ${nm} uc_nm)
 
-add_cdat_package_dependent(CMOR "" "" ON "CDAT_BUILD_CMOR" OFF)
+if (CDAT_BUILD_ALL)
```

Comment:
This whole `if` block can be replaced by 

`add_cdat_package_dependent(CMOR "" "" ${CDAT_BUILD_ALL} "CDAT_BUILD_CMOR" OFF)`